src/Microsoft.AspNet.Mvc.Core/ActionResults/ObjectResult.cs

```diff
@@ -194,6 +213,17 @@ public ObjectResult(object value)
             return selectedFormatter;
         }
 
+        private bool HasWildCardMediaAndSubMediaType(MediaTypeWithQualityHeaderValue[] mediaTypes)
```

Comment:
make static public, also check with @Tratcher to see if we can combine the work from strongly typed headers here (probably a follow up issue)